fhir-server/src/main/java/com/ibm/fhir/server/operation/FHIROperationRegistry.java

```diff
@@ -91,8 +91,10 @@ private boolean isValid(FHIROperation operation) throws FHIRValidationException,
         List<Issue> issues = FHIRValidator.validator().validate(opDef);
         if (!issues.isEmpty()) {
             for (Issue issue : issues) {
-                log.info("Issue: " + issue.getCode().getValue() + ":"
-            + issue.getSeverity().getValue() + ":" + issue.getDetails().getText().getValue());
+                if (log.isLoggable(Level.FINE)) {
+                    log.fine("Issue: " + issue.getCode().getValue() + ":"
```

Comment:
this is where the warning is squashed for the opDef?

Reply:
yes, you got it

Reply:
I was hoping that getting the up-to-date operation definitiions would resolve the issue, but it didn't.  So rather than patch the spec artifacts, I just toned down the logging.